platform/darwin/src/MGLShapeSource.h

```diff
@@ -41,6 +41,24 @@ FOUNDATION_EXTERN MGL_EXPORT const MGLShapeSourceOption MGLShapeSourceOptionClus
  */
 FOUNDATION_EXTERN MGL_EXPORT const MGLShapeSourceOption MGLShapeSourceOptionClusterRadius;
 
+/**
+ An `NSDictionary` object where the key is an `NSString`. The dictionary key will be the attribute feature attribute key. The resulting attribute value is aggregated from the clustered points. The dictionary value is an `NSArray` consisting of a reduce operator and a map expression.
+
+ The reduce operator is any `NSExpression` function that accepts at least two operands. You can use one of the following:
+
+ * An `NSString` containing a single function, such as `sum` or `max`.
+ * An `NSExpression` that takes two expression arguments: `featureAccumulated` and another valid expression.
+
+ The map expression is an `NSExpression` that produces the value of a single point within the cluster.
+
+ This option corresponds to the
+ <a href="https://www.mapbox.com/mapbox-gl-style-spec/#sources-geojson-clusterProperties"><code>clusterProperties</code></a>
+ source property in the Mapbox Style Specification.
+
+ This option only affects point features within an `MGLShapeSource` object; it
+ is ignored when creating an `MGLComputedShapeSource` object.
+ */
```

Comment:
Can we link to an example?

Reply:
We link to the clustering example a little further up, and I can update that to use clusterProperties. Would that work?